app/scripts/lib/strings.js

```diff
@@ -22,6 +22,7 @@ function () {
   // it's already being translated and may be used in the future.
   t('By proceeding, I agree to the <a id="service-tos" href="%(termsUri)s">Terms of Service</a> and' +
     '<a id="service-pp" href="%(privacyUri)s">Privacy Notice</a> of %(serviceName)s (%(serviceUri)s).');
+  t('Proceed');
```

Comment:
Do we need to Proceed here or have other strings?

We merged proceed here: https://github.com/mozilla/fxa-content-server/blob/a63443efa1abe7059bf1b74fbd1dfbc508dd79c2/app/scripts/templates/ready.mustache#L23